### PR TITLE
tech(command): include --directory option

### LINKED ISSUE #16: Directory option
Directory option - to setup the project in the given directory

### PR DESCRIPTION
**Pull request by shangan23**
#16 

- A new option added to the `react-chef ` command
- Restructure the command as per commander standard

To set up the project

~~npx react-chef -c <your-app-name>~~

```
npx react-chef <your-app-name>
```

To set up in the existing folder

~~npx react-chef -c <your-app-name> -d <relative-path>~~
```
npx react-chef <your-app-name> -d <relative-path>
```

### CODE REVIEW

File: recipes/index.js [hunk 1]
```diff
@@ -7,25 +7,26 @@ const {
...
 const install = require('./install');
 
-program.option("-f, --app-name <value>", "App Name");
-program.parse(process.argv);
+program.requiredOption("-c, --app-name <value>", "App Name");
+program.option("-d, --directory <value>", "Use directory");
+program.parse();
 
-//pre install check
-const appName = program.args[0];
```
Comment (ramsunvtech) [inline recipes/index.js:14]: this is needed to run the command without `-c`. should work in all below cases.

```
npx react-chef <your-app-name> // without c
npx react-chef <your-app-name> -d <relative-path> // without c
npx react-chef -c <your-app-name>
npx react-chef -c <your-app-name> -d <relative-path>
```

**Comment by shangan23** [comment]
***@***.****  - probably the option is not supported with commander need
to think about it. do you have any idea?

File: recipes/index.js (hunk 1 above)
Comment (shangan23) [reply recipes/index.js:14]: probably the option is not supported with commander need
to think about it. do you have any idea?

Comment (shangan23) [reply recipes/index.js:14]: resolved

File: recipes/install.js [hunk 2]
```diff
@@ -132,8 +135,12 @@ const install = function(appName) {
       return inquirer.prompt(projectQuestions);
     })
     .then((answers) => {
-      shell.mkdir(baseDirPath);
-      shell.cd(appName);
+      if(directory === undefined) {
```
Comment (ramsunvtech) [inline recipes/install.js]: keep it as `if(!directory) {

File: recipes/index.js [hunk 3]
```diff
@@ -7,11 +7,11 @@ const {
 } = require("./utils");
 const install = require('./install');
 
-program.option("-f, --app-name <value>", "App Name");
-program.parse(process.argv);
+program.option("-d, --directory <value>", "Use directory");
+program.parse();
```
Comment (ramsunvtech) [inline recipes/index.js:11]: will below case work?

```
npx react-chef appName
```

Comment (shangan23) [reply recipes/index.js:11]: @ramsunvtech - yes it will work

File: recipes/install.js (hunk 2 above)
Comment (shangan23) [reply recipes/install.js]: done

**Comment by ramsunvtech** [comment]
lgtm 👍

**Review by ramsunvtech — APPROVED** [review]

**Pull request merged**